compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -298,6 +290,17 @@ internal class ComposeContainer(
     }
 
     private fun setContent(content: @Composable () -> Unit) {
+        val mediator = mediator ?: ComposeSceneMediator(
```

Comment:
I suggest to move creation of it to initialization time, like was before this PR

Reply:
We can do it later. For now better to leave it as nullable.

Reply:
my comment is not about nullability, it's about initialization time